### ISSUE TITLE
[a11y design review]: Health Conditions domain

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#55624]
## Auditor(s)

Sara Smith

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [x] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [x] WCAG 2.1 AA
  - [x] WCAG 2.1 AAA
  - [x] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [x] High Contrast Mode
- [x] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav)
  - [x] Annotations for Tab order, reflow and reading order (if needed)
- [x] Accessible Name Annotations
  - [x] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [x] Text alternative for complex images such as charts graphs maps etc.
- [x] Language (ease of reading)
- [x] HTML Tags
  - [x] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [x] [Heading order](https://accessibility.18f.gov/headings/)
   - [x] Annotations (if needed)
  - [x] Semantic Consideration for paragraphs ordered lists numbering
  - [x] [Tables (heading rows)](https://design.va.gov/components/table)
- [x] Size of clickable areas

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

## Prototype Link

 - [Health conditions list view ](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/39DBAE50-6487-4084-8BBD-8E41A286BEDB)
  - [Details view](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/E40475C0-B20C-4249-B52D-C4F083040FAB)

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- If no research has been conducted with assistive technology users, it should be a priority

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

## Related Ticket/Epic
[MHV Accessibility Patterns Consolidation for Midpoint #56578](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/56578)

**Comment by sara-amanda** [comment]
@coultonbunney-usds  did we find out how this data is coming to us?

**Comment by SarahKay8** [comment]
No further design feedback to add for Health Conditions Domain. Closing @coultonbunney-usds @coforma-terry
